app/views/components/blockEditor.html

```diff
@@ -204,19 +204,86 @@
                 <span class="help-block help-block-hidden" ng-if="ruleValues(rule.parentBlockId).min && !ruleValues(rule.parentBlockId).max" translate>Please enter a number greater than or equal to {{ruleValues(rule.parentBlockId).min}}</span>
                 <span class="help-block help-block-hidden" ng-if="!ruleValues(rule.parentBlockId).min && ruleValues(rule.parentBlockId).max" translate>Please enter a number less than or equal to {{ruleValues(rule.parentBlockId).max}}</span>
               </div>
-              <select ng-switch-when="select" ng-model="rule.value" ng-options="v for v in ruleValues(rule.parentBlockId)" class="form-control"></select>
-              <select ng-switch-when="gender" ng-model="rule.value" class="form-control">
+              <select ng-switch-when="select" show-errors ng-model="rule.value" ng-options="v for v in ruleValues(rule.parentBlockId)" class="form-control"></select>
+              <select ng-switch-when="gender" show-errors ng-model="rule.value" class="form-control">
                 <option value="M" translate>Male</option>
                 <option value="F" translate>Female</option>
               </select>
-              <pick-a-date ng-switch-when="date" ng-model="rule.value" picker-min-date="getRangeValues(rule.parentBlockId).min" picker-max-date="getRangeValues(rule.parentBlockId).max"></pick-a-date>
+              <pick-a-date ng-switch-when="date" show-errors ng-model="rule.value" picker-min-date="getRangeValues(rule.parentBlockId).min" picker-max-date="getRangeValues(rule.parentBlockId).max"></pick-a-date>
             </div>
             <div class="clearfix visible-sm-block"></div>
             <div class="col-md-1 text-right">
               <button class="btn btn-danger" ng-click="removeRule(rule.id)" title="Delete Rule"><i class="fa fa-trash-o"></i></button>
             </div>
           </div>
         </div>
+
+        <div ng-if="block.type === 'checkboxQuestion' && disableForceSelectionRule() === false">
+          <p class="text-muted"><i class="fa fa-info-circle"></i>
+            <translate>This Rules section allows for conditional enforcement of any answer marked 'Force Selection'.</translate>
+            <translate>Rules can depend on questions of</translate> <a href="" tooltip="{{'Multiple Choice, Dropdown, Number, Date, Gender or Year in School' | translate}}"
+              translate>these types</a>
+            <translate>that are above this question.</translate>
+          </p>
+          <div class="form-horizontal">
+            <div class="form-group">
+              <div class="col-md-2">
+                <button class="btn btn-success" ng-click="addForceRule()"><i class="fa fa-plus"></i> <translate>Add Rule</translate></button>
+              </div>
+              <div class="col-md-8 rule-message-alignment">
+                <span class="pull-left" translate>Show question if:</span>
```

Comment:
I think it is just hard to quickly scan and know what section is which. Currently it looks like:
![screen shot 2016-10-10 at 11 18 05 pm](https://cloud.githubusercontent.com/assets/756501/19260607/d3b91e0c-8f3f-11e6-9a2f-e868f430b67c.png)

I was thinking having some bolder title would be nice. Smething like this maybe: (Using `h4`. Maybe the spacing could be smaller.)
![screen shot 2016-10-10 at 11 05 17 pm](https://cloud.githubusercontent.com/assets/756501/19260490/d88fbb30-8f3e-11e6-97a5-667b323d4092.png)

I was also wondering if bringing the radios up to the top would work since they are really what the action here is. Idk that I've got it looking exactly how I want it but here's an idea:
![screen shot 2016-10-10 at 11 46 34 pm](https://cloud.githubusercontent.com/assets/756501/19261102/d0ea2ece-8f43-11e6-953c-474de88038a9.png)

I was trying to consolidate the help message. Maybe something like this would be good:

> Rules are a great way to show questions or force selections based on the registrant's answers to other questions. Rules can depend on questions of these types that are asked before this question.

You could hide the `or force selections` part if it is not a checkbox question.

I was just playing around with it in the chrome inspector but here's my code from moving it around.
```html
<div class="tab-pane ng-scope active" ng-repeat="tab in tabs" ng-class="{active: tab.active}" tab-content-transclude="tab">
    
        <p class="text-muted ng-scope"><i class="fa fa-info-circle"></i>Rules are a great way to show questions or force selections based on the registrant's answers to other questions. Rules can depend on questions of <a href="" tooltip="Multiple Choice, Dropdown, Number, Date, Gender or Year in School" translate="">these types</a> that are asked before this question.</p>
        <div class="clearfix rule-message-alignment">
              <h4 class="pull-left" translate=""><span class="ng-scope">Show question if:</span></h4>
              <!-- ngRepeat: choice in ['AND','OR'] --><div ng-repeat="choice in ['AND','OR']" class="pull-left margin-left-10 radio padding-top-0 ng-scope">
                <label>
                  <input type="radio" show-errors="" ng-model="block.content.ruleoperand" ng-value="choice" class="ng-pristine ng-untouched ng-valid" name="224" value="AND">
                  <!-- ngIf: choice==='AND' --><strong ng-if="choice==='AND'" translate="" class="ng-scope"><span class="ng-scope">All rules are met</span></strong><!-- end ngIf: choice==='AND' -->
                  <!-- ngIf: choice==='OR' -->
                </label>
              </div><!-- end ngRepeat: choice in ['AND','OR'] --><div ng-repeat="choice in ['AND','OR']" class="pull-left margin-left-10 radio padding-top-0 ng-scope">
                <label>
                  <input type="radio" show-errors="" ng-model="block.content.ruleoperand" ng-value="choice" class="ng-pristine ng-untouched ng-valid" name="226" value="OR">
                  <!-- ngIf: choice==='AND' -->
                  <!-- ngIf: choice==='OR' --><strong ng-if="choice==='OR'" translate="" class="ng-scope"><span class="ng-scope">Any rules are met</span></strong><!-- end ngIf: choice==='OR' -->
                </label>
              </div><!-- end ngRepeat: choice in ['AND','OR'] -->
            <button class="btn btn-success pull-right" ng-click="addRule()"><i class="fa fa-plus"></i> <translate><span class="ng-scope">Add Rule</span></translate></button></div>
```